### PR TITLE
comment count break fix

### LINKED ISSUE #1483: Spacing of "Comments | XX" in bottom right of discussion container is off
Subject: Spacing of "Comments | XX" in bottom right of discussion container is off
Description:
See screenshot below

Results:
The word "Comments" and the digit should be on the same line separated by a pipe. 

Eg:    Comments | 3

Backend version: develop-d6943b34
Frontend version: develop-d6943b34
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:89.0) Gecko/20100101 Firefox/89.0
Page: https://app.couchers.org/community/4/americas
User (spoofable): jesse (57)

### PR DESCRIPTION
**Pull request by sadikekin**
closes #1483 

**Frontend checklist**
- [X] Formatted my code with `yarn format && yarn lint --fix`
- [X] There are no warnings from `yarn lint`
- [X] There are no console warnings when running the app
- [X] Added any new components to storybook
- [X] Added tests where relevant
- [X] All tests pass
- [X] Clicked around my changes running locally and it works
- [X] Checked Desktop, Mobile and Tablet screen sizes

Screenshot 2021-06-27 at 16 52 15

### CODE REVIEW

**Comment by lucaslcode** [comment]
Hi @sadikekin!
I think we should take this opportunity to make it look acceptable on mobile.
Right now, it has it's own column on the right and on narrow screens, the main text of the card gets really squished up
Can you move the comment count text so it is on it's own line underneath everything else? Also make sure the margins are suitable for the new position. And the count text can be the smaller "body2" instead of "body1" variant.

Now: 
image

Instead, something like this:
image

**Review by lucaslcode — CHANGES_REQUESTED** [review]
see above

**Comment by sadikekin** [comment]
Screenshot 2021-06-30 at 21 56 55

File: app/frontend/src/features/communities/discussions/DiscussionCard.tsx [hunk 1]
```diff
@@ -30,11 +30,12 @@ const useStyles = makeStyles((theme) => ({
     display: "flex",
     flexDirection: "column",
     flexGrow: 1,
-    marginInlineEnd: theme.spacing(2),
+    marginInlineEnd: theme.spacing(1),
```
Comment (lucaslcode) [inline app/frontend/src/features/communities/discussions/DiscussionCard.tsx]:
```suggestion
```

**Review by lucaslcode — CHANGES_REQUESTED** [review]
Looks great! I think we don't need this right margin at all now, since we have the card padding.

**Review by lucaslcode — APPROVED** [review]
Thanks!!

**Pull request merged**